_episodes/XX-access-data.md

```diff
@@ -0,0 +1,329 @@
+---
+
+A number of satellites take snapshots of the Earth's surface from space. The images recorded by these remote sensors
+represent a very precious data source for any activity that involves monitoring changes on Earth. Satellite imagery is
+typically provided in the form of geospatial raster data, with the measurements in each grid cell ("pixel") being
+associated to accurate geographic coordinate information.
+
+In this episode we will explore how to access open satellite data using Python. In particular,  we will
+consider [the Sentinel-2 data collection that is hosted on AWS](https://registry.opendata.aws/sentinel-2-l2a-cogs).
+This dataset consists of multi-band optical images acquired by the two satellites of
+[the Sentinel-2 mission](https://sentinel.esa.int/web/sentinel/missions/sentinel-2) and it is continuously updated with
+new images.
+
+# Search for satellite imagery
+
+## The SpatioTemporal Asset Catalog (STAC) specification
+
+Current sensor resolutions and satellite revisit periods are such that terabytes of data products are added daily to the
+corresponding collections. Such datasets cannot be made accessible to users via full-catalog download. Space agencies
+and other data providers often offer access to their data catalogs through interactive graphical user interfaces (GUIs,
+see for instance the [Copernicus Open Access Hub portal](https://scihub.copernicus.eu/dhus/#/home) for the Sentinel
+missions). Accessing data via a GUI is a nice way to explore a catalog and get familiar with its content, but it
+represents a heavy and error-prone task that should be avoided if carried out systematically to retrieve data.
+
+A service that offers programmatic access to the data enables users to reach the desired data in a more reliable,
+scalable and reproducible manner. An important element in the software interface exposed to the users (generally called
+the application programming interface, API) is the use of standards. Standards, in fact, can significantly facilitate
+the reusability of tools and scripts across datasets and applications.
+
+The SpatioTemporal Asset Catalog (STAC) specification is an emerging standard for describing geospatial data. By
+organizing metadata in a form that adheres to the STAC specifications, data providers make it possible for users to
+access data from different missions, instruments and collections using the same set of tools.
+
+> ## More Resources on STAC
+>
+> * [The STAC specification](https://github.com/radiantearth/stac-spec)
+> * [Tools based on STAC](https://stacindex.org/ecosystem)
+> * [STAC catalogs](https://stacindex.org/catalogs)
+{: .callout}
+
+## Search a STAC catalog
+
+The Sentinel-2 collection hosted on AWS is indexed in a STAC catalog that is accessible at the following link:
+
+~~~
+api_url = "https://earth-search.aws.element84.com/v0"
+~~~
+{: .language-python}
+
+> ## Exercise: Discover a STAC catalog
```

Comment:
before this exercise I think we can show an image of the radiant earth stac browser to give people a visual of what information a STAC catalog contains. Looking at the lesson webpage, it's a dense in the amount of text before the first image so I think this will make the first part of the lesson more engaging for someone who is browsing the lesson material or seeking out guidance on STAC.

Reply:
I have added a figure - the best "composition" I could come up with.. If you have suggestions for improvements, let me know!